driver/src/main/java/org/neo4j/driver/internal/pool/StandardConnectionPool.java

```diff
@@ -71,25 +65,19 @@
      */
     private final ValidationStrategy<PooledConnection> connectionValidation;
 
-    private final int connectionsPerDatabase;
     private final Clock clock;
+    private final Config config;
 
-    public StandardConnectionPool( Logging logging )
-    {
-        this( loadConnectors( logging ) );
-    }
-
-    public StandardConnectionPool( Collection<Connector> conns )
+    public StandardConnectionPool( Config config )
```

Comment:
If you do create longish parameter lists, consider making them microtypes so that you do not get lost in ints and Strings